src/status_im/contexts/wallet/common/sheets/network_preferences/view.cljs

```diff
@@ -1,32 +1,14 @@
 (ns status-im.contexts.wallet.common.sheets.network-preferences.view
-  (:require [clojure.string :as string]
-            [quo.core :as quo]
+  (:require [quo.core :as quo]
             [quo.foundations.colors :as colors]
-            [quo.foundations.resources :as resources]
             [quo.theme :as quo.theme]
             [react-native.blur :as blur]
             [reagent.core :as reagent]
             [status-im.contexts.wallet.common.sheets.network-preferences.style :as style]
+            [status-im.contexts.wallet.common.utils :as utils]
             [utils.i18n :as i18n]
             [utils.re-frame :as rf]))
 
-
-(defn- make-network-item
-  [{:keys [network-name] :as _network}
-   {:keys [title color on-change network-preferences state blur?] :as _options}]
-  {:title        (or title (string/capitalize (name network-name)))
-   :blur?        blur?
-   :image        :icon-avatar
-   :image-props  {:icon (resources/get-network network-name)
-                  :size :size-20}
-   :action       :selector
-   :action-props {:type                (if (= :default state)
-                                         :filled-checkbox
-                                         :checkbox)
-                  :customization-color color
-                  :checked?            (contains? network-preferences network-name)
-                  :on-change           on-change}})
-
```

Comment:
Moved out this function to `utils` ns (with a minor addition) as the network preferences and network filter/selector UI have quo/category components.